sd-array/task_3.cpp

```diff
@@ -0,0 +1,52 @@
+#include <iostream>
+using namespace std;
+int main(){
+    const int MAX = 1024;
+    int n, a[MAX], k, gcd = 1;
+    do{
+        cin >> n >> k;
+        /*има ли по-оптимален начин за това?*/
+        for(int i = 1; i <= n && i <=k; i++){
+            if(n % i == 0 && k % i == 0){
+                gcd = i;
+            }
+        }
```

Comment:
Има по-добър начин за намиране на GCD.
```suggestion
        int x = n, y = k;
        while(x != y){
            if (x < y) swap(x, y);
            x -= y; 
        }
        gcd = x;
```